test/integration/consul-container/upgrade/version_oss.go

```diff
@@ -1,9 +0,0 @@
-//go:build !consulent
```

Comment:
Oh I see you are moving these under a `utils` package -- I think that's a great idea!

For this kind of refactor, you will need to make an "extra" commit to fix ENT specific changes since it will break when it carries over to our ENT repo (you can browse the repo to see that there's ent specific functions there).

So, I think this is fine to do, just something to keep in mind as you get to merge this in that  we will need to fix in ENT.

Alternatively, you could factor out these changes (the refactor into `utils`) and follow up in a subsequent PR from the ENT side. Refactoring is usually easier from that side.